src/app/helper/colourHelper.js

```diff
@@ -179,6 +179,8 @@ helper = {
             "rgb": parseRgb(color.toRgb(), "string"),
```

Comment:
I reckon these helper methods could be streamlined theres a lot of duplicated code perhaps something like
```
    const getColourObject: (color) => {
          return {
            "rgb": parseRgb(color.toRgb(), "string"),
            "hex": color.toHex(),
            "hue": parseDecimal(color.toHsl()["h"]),
            "rgbOpacity": 1,
            "hexOpacity": 1,
            "saturation": parseDecimal(color.toHsl()["s"]),
            "lightness": parseDecimal(color.toHsl()["l"]),
            "theme": color.isDark() ? "light" : "dark",
            "bgColour": color.toHex()
        };
    };
   colourHelper = {
    .....
    randomise: () =>  getColourObject(tinycolor.random()),

    lighten: (hex) =>  getColourObject(tinycolor(hex).lighten()),
    
    darken: (hex) => getColourObject(tinycolor(hex).darken()),
   ......     
}```